nixos/modules/services/network-filesystems/iroh.nix

```diff
@@ -0,0 +1,275 @@
+{ config, lib, pkgs, utils, ... }:
+with lib;
```

Comment:
many people are highly discouraging this these days

Reply:
I started with a copy/paste from elsewhere (mea culpa), seems as though I may have picked up some anachronisms. Is there a linter or repository of heuristics for good module development that you happen to know of?

Reply:
@SuperSandro2000 is the best `nix` linter that i know of 😉